cdhweb/pages/tests/test_hooks.py

```diff
@@ -28,3 +28,23 @@ def test_create_redirect_on_slug_change(db, admin_client, content_page):
         assert Redirect.objects.filter(
             old_path=old_path, redirect_page=content_page
         ).exists()
+
+    def test_create_redirect_on_move_page(db, admin_client, content_page, homepage):
+        """redirect should automatically be created on page move"""
+        assert Redirect.objects.count() == 0
+        # Content page starts out underneath a landing page; we move it to
+        # live underneath the homepage instead: /landing/content -> /content
+        old_path = content_page.url[:-1]
```

Comment:
Is the `[:1]` for the trailing slash? seems weird that that's part of the url but not part of the redirect (although I have noticed that when I've used the redirects manually)

Reply:
I think that's right, but it's weird to me too. that's how the wagtail recipe has it 🤷🏻